File: mycode/java-me/calculater2.java
import java.util.*;
public class calculater2 {
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        System.out.print("enter 1st no=");
        int a=sc.nextInt();
        System.out.print("enter 2nd no=");
        int b=sc.nextInt();
        System.out.print("enter the operater=");
        char ch=sc.next().charAt(0);
        switch(ch){
            case '+':
            System.out.printf("the sum of your no=%d",a+b);
            break;
            case '-':
            System.out.printf("subtract of your no=%d",a-b);
            break;
            default :
            System.out.println("invalid");
        }

    
    }
    
}

File: mycode/c++me/calculater2.cpp
#include<iostream>
using namespace std;
int main(){
int n1,n2;
char op;
cout<<"enter 1st no=";
cin>>n1;
cout<<"enter 2nd no=";
cin>>n2;
cout<<"enter operater=";
cin>>op;
switch (op){
    case '+':
    cout<<"the sum ur no="<<n1+n2<<endl;
    break;
    case '-':
    cout<<"the subtract no ur no="<<n1-n2<<endl;
    break;
    case '*':
    cout<<"the multiple or ur no="<<n1*n2<<endl;
    break;
    case '/':
    cout<<"the divide or ur no="<<n1/n2<<endl;
    break;
}


return 0;
}

